src/main/resources/templates/fragments/inputs/date.html

```diff
@@ -0,0 +1,81 @@
+<th:block
+    th:fragment="date"
+    th:with="
+      hasLabel=${!#strings.isEmpty(label)},
+      hasAriaLabel=${!#strings.isEmpty(ariaLabel)},
+      hasErrorDate=${
+        errorMessages != null &&
+        errorMessages.get(inputName) != null &&
+        #arrays.length(errorMessages.get(inputName)) > 0
+      },
+      hasErrorMonth=${
```

Comment:
I don't think these are needed now ? Can remove them from the template

Reply:
looks like theyre used for `aria-invalid`